### DISCUSSION TITLE
Sort City Names options alphabetically

### DISCUSSION THREAD

**Discussion by pookage** [voxeltycoon/issues#422] [UI&UX]
**Describe the bug**
The default 'City Names' option is 'American', but 'American' does not exist in the dropdown to select; even when another option is selected.

![image](https://user-images.githubusercontent.com/3370135/115118247-29343780-9f9a-11eb-877b-5c2ce5d0ea97.png)

**Steps to reproduce**
1. Start a new game
2. Open the 'city names' dropdown
3. Note the absence of 'American' (side-note: also maybe include 'English' for us non-americans?)

**Build version**
`0.85.0.24 steam`

**Platform**
Windows 10, Steam

**Reproducibility**
Every time.

**Logs/saves**
N/A

**Comment by pookage** [comment]
...nevermind! I just saw it was between German and Spanish 🤦  So I guess the ticket should be that it's not alphabetised correctly, ha. I'll leave this open in case someone wants to convert the ticket, but happy for it to be closed too (although, I would REALLY like there to be an 'English' option here too - kinda weird there being Australian and American but not English 😢 )

**Comment by victorleblais** [comment]
I'm converting that issue to a suggestion ;)
